input/cql/COVID19EmergencyDeptAssessment.cql

```diff
@@ -666,9 +666,14 @@ define "Risk Factor Components": List {
     otherwise returns the count of non-null risk factors.
 */
 define "Risk Factors count":
-    allRiskFactors("All Active Conditions")
+  if "All Risk Factors Are Null" then  null else
+  allRiskFactors("All Active Conditions") 
 
 
+define "All Risk Factors Are Null":
+  if "Risk Factor Components" contains false
+  or "Risk Factor Components" contains true then false
```

Comment:
Not sure if calling "Risk Factor Components" twice will re-evaluate the List and cause any performance issue. Maybe the list can be assigned a variable?

Reply:
Don't see any performance lagging on local. We will keep an eye out for performance in staging. Could not find a way to assign as a List - not avail in CQL.